joomla/joomla-cms
# Hidden fieldsets

#### Summary of Changes

If you create a repeatable field (via e.g. a plugin) and add it to differnt components (e.g. com_users), the fieldsets from the field are displayed as empty tabs
#### Testing Instructions

1, Add the XML code for a repeatable field to administrator/components/com_users/models/forms/user.xml (or write a plugin which loads a differnt XML into the user form). See here for the code: https://docs.joomla.org/Repeatable_form_field_type
2. Open the user component and create a new user
3. a "broken" tab should appear
4. Apply patch => try again
